contracts/AdminList.sol

```diff
@@ -0,0 +1,93 @@
+pragma solidity >=0.4.22 <0.6.0;
+
+import "solidity-linked-list/contracts/StructuredLinkedList.sol";
+
+
+contract AdminList {
+    using StructuredLinkedList for StructuredLinkedList.List;
+
+    StructuredLinkedList.List private list;
+
+    function size() internal view returns (uint256) {
+        return list.sizeOf();
+    }
+
+    function exists(address _address) internal view returns (bool) {
+        return list.nodeExists(uint256(_address));
+    }
+
+    function add(address _address) internal returns (bool) {
+        return list.push(uint(_address), true);
+    }
+
+    function remove(address _address) internal returns (bool) {
+        uint node = uint(_address);
+        return list.remove(node) != 0 ? true : false;
+    }
+
+    function get(uint index) internal view returns (bool _exists, address _address) {
+        uint listSize = list.sizeOf();
+        if (index >= listSize) {
+            return (false, address(0));
+        }
+
+        uint counter = 0;
+        uint pointer = 0;
+        bool hasFound = false;
+
+        while(counter <= listSize) {
+            (bool nodeExists, uint256 prev, uint256 next) = list.getNode(pointer);
+            if (nodeExists) {
+                if (counter == index + 1) {
+                    hasFound = true;
+                    break;
+                } else {
+                    counter++;
+                    pointer = prev;
+                }
+            } else {
+                break;
+            }
+            //Getting rid of unused variable warning
```

Comment:
I'm guessing it's the linter that's throwing this from line 39? Better to disable that linting with `/* solium-disable-next-line */`

Reply:
It is actually the compiler